sql-bricks.js

```diff
@@ -2,7 +2,7 @@
   "use strict";
 
   var is_common_js = typeof exports != 'undefined';
-  var default_opts = { placeholder: '?' };
+  var default_opts = { placeholder: '$%d' };

```

Comment:
@vitalif: Thanks! This variable should also be used where we have an existing `{'placeholder': '$%d'}` (in `Statement.prototype.toParams` and `Statement.prototype.toString`).